### DISCUSSION TITLE
window.state("zoomed") not maximizing the window

### DISCUSSION THREAD

**Discussion by HyperNylium** [TomSchimansky/CustomTkinter#1819] [Q&A] [answered]
So, i have been having this problem where in this simple script:
```python
import customtkinter

window = customtkinter.CTk()

window.state("zoomed")

window.mainloop()
```
the window seems to zoom in (go fullscreen) but then goes back to normal. I did take a video of what was happening on my end. In the video i changed my resolution in scale factor, still no luck. Don't really know if that means anything but tryed anyways ㄟ( ▔, ▔ )ㄏ

The goal here is that when i call `window.state("zoomed")` it maximizes the window and vice versa

any help would be appreciated :)

https://github.com/TomSchimansky/CustomTkinter/assets/96999894/02d4d54c-7ff8-4c15-a901-72c00004e745

**Comment by Akascape** [comment, marked as answer]
@HyperNylium 
```python
from customtkinter import CTk

root = CTk()
root.after(50, lambda: root.state('zoomed'))
root.mainloop()
```

**Reply by HyperNylium** [reply]
This worked, thank you!
Could you explain why we have to wait 50ms then run the `root.state()` instead of just putting it there without the timeout?
I come from tkinter and didn't have to do this so was just curious :)

**Reply by Akascape** [reply]
Customtkinter just blocks some of these commands by setting internal 'after' methods.  Like you can try `root.iconbitmap` on a toplevel and it will not change the icon, but if you use 'after' method then it will work. These are actually some bugs not fixed yet.

**Reply by HyperNylium** [reply]
Oh ok. Thank you for the explanation and your time. I really appreciated the help. Have a good evening.
